zoe-core/src/functions/admin.kt

```diff
@@ -362,15 +400,41 @@ data class DeploySchemaConfig(
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
 @JsonSubTypes(
-    JsonSubTypes.Type(value = ActualRunResponse::class, name = "actual"),
-    JsonSubTypes.Type(value = DryRunResponse::class, name = "dry")
+    JsonSubTypes.Type(value = DeploySchemaResponse.ActualRunResponse::class, name = "actual"),
+    JsonSubTypes.Type(value = DeploySchemaResponse.DryRunResponse::class, name = "dry")
 )
 sealed class DeploySchemaResponse {
     data class ActualRunResponse(val id: Int, val subject: String) : DeploySchemaResponse()
     data class DryRunResponse(val subject: String, val schema: String, val registry: String) :
         DeploySchemaResponse()
 }
 
+data class DeleteSchemaConfig(
+    val registry: String,
+    val props: Map<String, String?>,
+    val subject: String,
+    val schemaVersion: Int?,
+    val hardDelete: Boolean,
+    val dryRun: Boolean
+)
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
+@JsonSubTypes(
+    JsonSubTypes.Type(value = DeleteSchemaResponse.ActualRunResponse::class, name = "actual"),
+    JsonSubTypes.Type(value = DeleteSchemaResponse.DryRunResponse::class, name = "dry")
+)
+sealed class DeleteSchemaResponse {
+    data class ActualRunResponse(
+        val response: List<Int>,
```

Comment:
I think it's clearer to name this attribute `deletedSubjectIds` or `deletedVersions` whatever this is instead of response?

Reply:
Indeed I missed that, good one!

Reply:
Should be fixed, I rebased the commits to keep them clean as I wasn't sure if you were squashing before merging.

Reply:
Perfect 👍🏼